catroweb.yaml

```diff
@@ -694,6 +695,8 @@ components:
           - most_downloaded
           - example
           - scratch
+          - more_from_this_user
```

Comment:
projects/user  already exists and should do the same

only difference, the 1 project (here given as id) may is already removed from the results?

@HCrane